compact/hcal/lfhcal_insertGeom.xml

```diff
@@ -0,0 +1,206 @@
+<lccdd>
+
+  <info name="lfhcal.xml"
```

Comment:
The insert geometry should not duplicate the geometry of the geometry without insert. This makes code management difficult. If possible, the insert geometry should be separate, with module geometry definitions in a separate file that can be read first. The insert geometry should only define what is different/additional for the insert geometry.

(I won't review the rest of this file since the same comments as for lfhcal.xml apply.)

Reply:
this is not the insert Geometry but the lfhcal geometry in case there is an insert, which means I need the positions to be different as I need to take out modules which otherwise would collide with the actual insert.